test/jdk/java/net/httpclient/AggregateRequestBodyTest.java

```diff
@@ -431,6 +428,11 @@ public void onSubscribe(Subscription subscription) {
         @Override
         public void onNext(ByteBuffer item) {
             items.addLast(item);
+            int available = semaphore.availablePermits();
+            if (semaphore.availablePermits() > Integer.MAX_VALUE - 8) {
```

Comment:
In context of reporting the `available` value in the exception message thta gets thrown in the next line, should this check have used the local variable `available` or does it not matter?

Reply:
Oh! My mistake. I introduced the variable specifically for this, but didn't update all the places. Thanks for noticing. Not that it matters much, we won't reach Integer.MAX_VALUE - 8 in this test anyway :-)

Reply:
Fixed in last commit.